schema/std-priv-extensions.asn

```diff
@@ -0,0 +1,299 @@
+H-Extension -- H: Hypervisor
+DEFINITIONS
+    AUTOMATIC TAGS ::=
+BEGIN
+    EXPORTS H;
+
+    H ::= SEQUENCE {
+        version Version OPTIONAL,
+        ...
+    }
+
+END -- H-Extension
+
+Shcounterenw-Extension -- Shcounterenw
+DEFINITIONS
+    AUTOMATIC TAGS ::=
+BEGIN
```

Comment:
Is it even allowed to have multiple DEFINITIONS in a single file like that? How does the IMPORTS statement know which file to look for in this case?

I'm a little on the fence as to whether it makes sense to put all stdpriv extensions into a single file. Having one file per extension feels a little excessive, since most of them will be place holders. But all in one will lead to an ever-growing file. Ideally we can split these up into functional areas, or something. I'm not sure where to draw the line. This is likely a case where no solution is perfect.

Reply:
The IMPORTS statement doesn't look in files. It looks for modules. The tools find all the .asn files and parse them all. Then the IMPORTS statement refers to the modules defined in those files.